controller/controller.js

```diff
@@ -0,0 +1,17 @@
+// c = controller, m = model, v = view, db = database
+const m = require("../model/model");
+
+// TODO: find out how error catching is meant to be, repeating .catch here seems not very DRY
+
+// /api/topics Endpoint
+exports.getApiTopics = (request, response) => {
+    m.selectTopics().then((selectTopicsQueryResult) => {
+        const responseObject = {topics: selectTopicsQueryResult.rows};
```

Comment:
The controller should just been handling the req and res. Any other manipulation should be performed in your model